scripts/sample-reads-randomly.py

```diff
@@ -127,7 +130,8 @@ def main():
     # read through all the sequences and load/resample the reservoir
     for filename in args.filenames:
         print >>sys.stderr, 'opening', filename, 'for reading'
-        for record in screed.open(filename, parse_description=False):
+        screed_iter = screed.open(filename, parse_description=False)
+        for record in broken_paired_reader(screed_iter):
```

Comment:
I personally dislike treating 'record' as opaque here -- and throughout the code base, 'record' is used as a single screed record. so would suggest breaking this out into _, is_paired, record1, record2.